repos/system_upgrade/el7toel8/actors/kernel/checkkerneldrivers/actor.py

```diff
@@ -52,29 +51,17 @@ def process(self):
                     if active_module.filename in removed_drivers:
                         collected_drivers.add(active_module.filename)
 
-            # Going over the collected drivers and considering for reporting only
-            # those drivers that are currently used by some device.
-            udevadm_db = ''
-            for fact in self.consume(UdevAdmInfoData):
-                udevadm_db += fact.db
-            for line in udevadm_db.split('\n'):
-                if 'E: DRIVER=' in line:
-                    _, driver = line.split('=')
-                    if driver in collected_drivers:
-                        drivers_to_report.add(driver)
-
             # In the end, we are only going to report drivers that are:
             # - removed in the RHEL8 (are part of files/removed_drivers.txt)
             # - not whitelisted
-            # - currently being used by some device
-            if drivers_to_report:
+            if collected_drivers:
                 title = ('Detected loaded kernel drivers which have been removed '
                          'in RHEL 8. Upgrade cannot proceed.')
                 URL = ('https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/'
                        'considerations_in_adopting_rhel_8/index#removed-device-drivers_hardware-enablement')
-                summary = ('Support for the following currently loaded RHEL 7 '
+                summary = ('Support for the following RHEL 7 '
                            'device drivers has been removed in RHEL 8: \n     - {}'
-                           '\nPlease see {} for details.'.format('\n     - '.join(drivers_to_report), URL))
+                           '\nPlease see {} for details.'.format('\n     - '.join(collected_drivers), URL))
                 remediation = ('Please disable detected kernel drivers in '
                                'order to proceed with the upgrade process using the rmmod tool.')
```

Comment:
Stupid question - rmmod tool only? Can't we do the same with modprobe -r?

Reply:
Yes we can. I will update the message. Thanks